python/docs/source/user_guide/pandas_on_spark/supported_pandas_api.rst

```diff
@@ -0,0 +1,1320 @@
+| floordiv              | O        | ``fill_value``, ``level``                                    |
++-----------------------+----------+--------------------------------------------------------------+
+| ge                    | O        | ``fill_value``, ``level``                                    |
++-----------------------+----------+--------------------------------------------------------------+
+| get                   | O        |                                                              |
++-----------------------+----------+--------------------------------------------------------------+
+| groupby               | O        |                                                              |
++-----------------------+----------+--------------------------------------------------------------+
+| gt                    | O        | ``fill_value``, ``level``                                    |
++-----------------------+----------+--------------------------------------------------------------+
+| hasnans               | O        |                                                              |
++-----------------------+----------+--------------------------------------------------------------+
+| head                  | O        |                                                              |
++-----------------------+----------+--------------------------------------------------------------+
+| hist                  | O        |                                                              |
++-----------------------+----------+--------------------------------------------------------------+
+| iat                   | O        |                                                              |
++-----------------------+----------+--------------------------------------------------------------+
+| idxmax                | O        | ``axis``                                                     |
++-----------------------+----------+--------------------------------------------------------------+
+| idxmin                | O        | ``axis``                                                     |
++-----------------------+----------+--------------------------------------------------------------+
+| iloc                  | O        |                                                              |
++-----------------------+----------+--------------------------------------------------------------+
+| index                 | O        |                                                              |
++-----------------------+----------+--------------------------------------------------------------+
+| infer_objects         | X        |                                                              |
++-----------------------+----------+--------------------------------------------------------------+
+| interpolate           | X        |                                                              |
```

Comment:
Let's keep the PR up to date. e.g.) this is implemented https://github.com/apache/spark/commit/30fc0ba2307dcb71de1ebf785b96b4e5f05d96bd

Reply:
Seems like `interpolate` and `ewm` will be supported from 3.4.0.

Maybe can we separate the PR for 3.3.0 and master ?

Reply:
Yeah, let's fix it here, and create another PR to remove it out for branch-3.3.

Reply:
Sounds good